.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Discussions
+    url: https://github.com/huggingface/datasets/discussions
+    about: Please ask and answer questions here, and engage with other community members
```

Comment:
Maybe adding additionally/instead a link to the Forum? https://discuss.huggingface.co/

Reply:
@theo-m @lhoestq do you know which is our preferred channel for Discussions? 
- https://github.com/huggingface/datasets/discussions
- https://discuss.huggingface.co/

Once this is clear, I will merge this PR :)

Reply:
@lhoestq maybe we should keep accepting questions as GH Issues? 

Otherwise, we should regularly keep track of the questions posted in the Discussion forums, while we are already doing this for GH Issues.

In that case, I would create a template for Questions...

Reply:
We should definitely put the forum https://discuss.huggingface.co/ forward !